src/saving/Save.cs

```diff
@@ -152,6 +154,18 @@ public void SaveToFile()
         }
     }
 
+    internal static void LogErrorToFile(string saveName, string type, Exception ex)
```

Comment:
I don't think this is needed. I think it's better to include all the logging in just the main log file for easily asking the players to upload that somewhere.